packages/truffle-core/lib/commands/config.js

```diff
@@ -12,27 +117,94 @@ const command = {
         option: "--disable-analytics",
         description:
           "Disable Truffle's ability to send usage data to Google Analytics"
+      },
+      {
+        option: "enable",
+        discription: "Enable a Truffle Attribute"
```

Comment:
Should attribute be capitalized in all these cases?

Reply:
:man_shrugging: Y'all can change it to whatever you want

Reply:
@gnidan Can you weigh in on this one? What do you think about for all these we call them "configuration options". So for example, "Enable a Truffle configuration option"? Does that sound too robotic?